members/models.py

```diff
@@ -313,11 +313,16 @@ def save(self, *args, **kwargs):
         return super(ActivityParticipant, self).save(*args, **kwargs)
 
 class Volunteer(models.Model):
-    member = models.ForeignKey(Member)
+    class Meta:
+        verbose_name = "Frivillig"
+        verbose_name_plural = "Frivillige"
+    member = models.ForeignKey(Person)
     department = models.ForeignKey(Department)
     def has_certificate(self):
-        return self.person.has_certificate
-    added = models.DateTimeField(auto_now_add=True, blank=True, editable=False)
+        return self.member.has_certificate
```

Comment:
men det er en person. Members er noget andet (og på sigt, skal frivillige også være members pga. DUF midler) - så lad os omdøbe members -> person.